Dockerfile.tests

```diff
@@ -9,6 +9,7 @@ ARG LIBSLIRP_COMMIT
 RUN  git pull && git checkout ${LIBSLIRP_COMMIT} && meson setup build && ninja -C build install
 COPY . /slirp4netns
 WORKDIR /slirp4netns
+RUN chmod +x tests/*
```

Comment:
chmod +x should be done in the git commit itself, not here

Reply:
fixed